src/sqliteformat.cpp

```diff
@@ -52,12 +52,58 @@ class mKCal::SqliteFormat::Private
         : mStorage(storage), mDatabase(database)
         , mSelectCalProps(nullptr)
         , mInsertCalProps(nullptr)
+        , mSelectIncProperties(nullptr)
+        , mSelectIncAttendees(nullptr)
+        , mSelectIncAlarms(nullptr)
+        , mSelectIncRecursives(nullptr)
+        , mSelectIncRDates(nullptr)
+        , mSelectIncAttachments(nullptr)
+        , mSelectDeletedIncidence(nullptr)
+        , mDelIncComponents(nullptr)
```

Comment:
Sure, I've done like that:
- moved initialisation at declaration position,
- kept the `Inc` abbreviation but expand the Select, Insert etc.

I've kept the `CalProps` variant together though because they are dealing with notebook stuff while all others are dealing with incidences.